apps/frontend/src/lib/components/FeedbackButtons.svelte

```diff
@@ -22,22 +24,22 @@
 <div class="ml-2 flex gap-1 border-l border-gray-300 pl-2 dark:border-gray-600">
 	<Button
 		onclick={() => handleFeedback('up')}
-		class="p-1.5! {feedbackGiven === 'up' ? 'bg-gray-200 dark:bg-gray-700' : ''}"
-		color="alternative"
-		size="xs"
+		variant="ghost"
+		size="icon-sm"
+		class="h-6 w-6 p-1.5 {feedbackGiven === 'up' ? 'bg-gray-200 dark:bg-gray-700' : ''}"
 		title={m.message_feedback_good()}
 	>
```

Comment:
Nice, I see how this working. We could end up with just a FeedBackButton containing a button. Our entire frontend can be based on function components and we can think about user experience not classes.